modules/pipeline/providers/reconciler/taskrun/taskop/prepare.go

```diff
@@ -354,6 +354,11 @@ func (pre *prepare) makeTaskRun() (needRetry bool, err error) {
 	task.Extra.PublicEnvs["PIPELINE_REQUESTED_MEM"] = fmt.Sprintf("%g", task.Extra.RuntimeResource.Memory)
 	task.Extra.PublicEnvs["PIPELINE_REQUESTED_DISK"] = fmt.Sprintf("%g", task.Extra.RuntimeResource.Disk)
 
+	// edge pipeline envs
+	edgePipelineEnvs := pre.EdgeRegister.GetEdgePipelineEnvs()
```

Comment:
`GetEdgePipelineEnvs` should in edge_pipeline or edge_pipeline_register?

Reply:
in the feature, should impl in env-provider